packages/create-app/index.js

```diff
@@ -44,6 +46,14 @@ async function init() {
       message: `Project name:`,
       initial: 'vite-project'
     })
+    const validateResult = validatePackageName(name)
+    if (!validateResult.validForNewPackages) {
+      console.error(
+        // Only one name to be validated, so there will be only one error
+        validateResult.errors.length && validateResult.errors[0]
+      )
+      process.exit(1)
+    }
```

Comment:
```suggestion
    const { errors } = validatePackageName(name);
    if (errors) {
        errors.unshift(`Invalid package name: ${name}`);
        console.error(errors.join('\n  - '));
        process.exit(1);
    }
```

While only one name will be validated, multiple errors can in fact be returned. For example, the name `'foobar '` will result in both `name cannot contain leading or trailing spaces` and `name can only contain URL-friendly characters` being returned. Multiple errors.

Reply:
@rschristian Thank you. I correct it now.